diff --git a/scripts/index.js b/scripts/index.js
index d2d6a52c774ea776e76cf77ca793c5ab6c8d9a96..b38643d70577594778858bbdfb46aff9ec23e457 100755
--- a/scripts/index.js
+++ b/scripts/index.js
@@ -11519,8 +11519,8 @@ class BuildCommand extends Command {
                 .update(version$1)
                 .digest('hex')
                 .substring(0, 8);
-            const intermediateTypeFile = path$1.join(os.tmpdir(), `${cargoArtifactName}-${cwdSha}.napi_type_def.tmp`);
-            const intermediateWasiRegisterFile = path$1.join(os.tmpdir(), `${cargoArtifactName}-${cwdSha}.napi_wasi_register.tmp`);
+            const intermediateTypeFile = path$1.join(os.tmpdir(), `${cargoArtifactName}-${cwdSha}.${Date.now()}.napi_type_def.tmp`);
+            const intermediateWasiRegisterFile = path$1.join(os.tmpdir(), `${cargoArtifactName}-${cwdSha}.${Date.now()}.napi_wasi_register.tmp`);
             debug$8(`intermediate type def file: ${intermediateTypeFile}`);
             const commandEnv = Object.assign(Object.assign(Object.assign({}, process.env), additionalEnv), { TYPE_DEF_TMP_PATH: intermediateTypeFile, WASI_REGISTER_TMP_PATH: intermediateWasiRegisterFile, CARGO_CFG_NAPI_RS_CLI_VERSION: version$1 });
             try {